static/js/lightbox.js

```diff
@@ -66,27 +80,80 @@ export class PanZoomControl {
         });
     }
 
+    constrainImage(e) {
+        // Instead of using panzoom's built in bounds option which was buggy
+        // at the time of this writing, we act on pan/zoom events and move the
```

Comment:
Is there an upstream issue we can link to?

Reply:
Yes, added a link.